.storybook/themes/news.scss

```diff
@@ -687,7 +802,12 @@
 );
 
 @use "../../blocks/category-carousel-block";
-@use "../../blocks/header-block";
```

Comment:
shouldn't be removing this?

Reply:
Bad merge, added it back and also organized the imports alphabetically